react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -39,21 +40,24 @@ class Game extends Component {
     const moves = this.getMoves(history);
     const status = winner ? `Winner: ${winner}` : `Next player: ${xIsNext ? 'X' : 'O'}`;
     return (
-      <div className={styles.game}>
-        <div className={styles.gameBoard}>
-          <Board
-            squares={squares}
-            onClick={this.handleClick}
-          />
+      <Fragment>
+        <TopBar />
+        <div className={styles.game}>
+          <div className={styles.gameBoard}>
+            <Board
+              squares={squares}
+              onClick={this.handleClick}
+            />
+          </div>
+          <div className={styles.gameInfo}>
+            <div>{status}</div>
+            <ol>{moves}</ol>
+          </div>
+          <div className={styles.matchHistory}>
+            <MatchHistory matches={matchHistory} isLoading={isLoading} />
+          </div>
         </div>
-        <div className={styles.gameInfo}>
-          <div>{status}</div>
-          <ol>{moves}</ol>
-        </div>
-        <div className={styles.matchHistory}>
-          <MatchHistory matches={matchHistory} isLoading={isLoading} />
-        </div>
-      </div>
+      </Fragment>
```

Comment:
Just a quick note. In newer versions of react you can just write `<> .... </>` and it will work as Fragment too